bootstrap.sh

```diff
@@ -61,10 +61,10 @@ then
 fi
 
 # remove my.cnf and link our custom file if a custom my.cnf exists
-# if [ -e /vagrant/serverConfiguration/my.cnf ]
+# if [ -e /vagrant/serverConfiguration/vagrant.my.cnf ]
```

Comment:
why keeping vagrant, remove the parts for vagrant and keep it setup with docker.